test/cql-pytest/test_describe.py

```diff
@@ -0,0 +1,344 @@
+# Copyright 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+#############################################################################
+# Tests for server-side describe
+#############################################################################
+
+# Tests for listing describe statements compares the result from describe to
+# the corresponding table from `system_schema` keyspace. This way, the tests ensure
+# all of the elements are present in the describe listing.
```

Comment:
This paragraph isn't clear to me. Isn't it just an unclear version of the next paragraph? Or was it trying to say something different?

Reply:
Ok, after reading the code below, I understood you actually have **two** types of test - one is described by the paragraph above (these are tests which lists things, like tables, keyspaces, etc., once via "describe" and by once looking at system tables - and compare the two), and one is described by the paragraph below. Maybe this can be explained more clearly, including explicitly mentioning there are two types of tests, and having bullets or numbers saying these are the two types.

Reply:
I've rewrote the comment, I hope now it's more clear :)